constellation_base/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.dispatch import receiver
+from django.contrib.auth.signals import user_logged_in
+
+from .models import LocalGroupACL
+
+
+@receiver(user_logged_in)
+def addSupplementalGroups(sender, user, request, **kwargs):
+    """Apply additional groups on login"""
+    for group in list(LocalGroupACL.objects.all()):
```

Comment:
`list` is not needed...
`for group in LocalGroupACL.objects.all()` should work.
`for group in LocalGroupACL.objects.all().iterator()` will also work. (If I recall correctly)